src/tests/CampaignAdSchedules.test.ts

```diff
@@ -46,6 +48,48 @@ describe('Campaign Ad Schedules', async () => {
         criterion_id = new_criterion.id
         done()
     })
+    
+    it('Creates 2 New Campaign Ad Schedules', async done => {
+        expect.assertions(1)
+        
+        const new_criteria_config = [
+            {
+                campaign_id,
+                ad_schedule: {
+                    day_of_week: 'WEDNESDAY',
+                    start_hour: 10,
+                    end_hour: 18,
+                    start_minute: 'ZERO',
+                    end_minute: 'ZERO',
+                },
+            }, {
+                campaign_id,
+                ad_schedule: {
+                    day_of_week: 'THURSDAY',
+                    start_hour: 10,
+                    end_hour: 18,
+                    start_minute: 'ZERO',
+                    end_minute: 'ZERO',
+                },
+            }
+        ]
+        
+        const new_criteria = await customer.campaignAdSchedules.create(new_criteria_config)
+
+        expect(new_criteria).toEqual({
+            id: expect.any(String),
+            resource_name: expect.any(String),
+        })
```

Comment:
I just updated a method which maps results to support multiple rows for the batch operation, so `new_criteria` is going to be an array here. We need to change this assertion to e.g.:  
```
expect(new_criteria).toContainEqual(
    expect.objectContaining({
        id: expect.any(String),
        resource_name: expect.any(String),
     })
)
```
This applies to all added tests.

Reply:
I believe I got all of these